### PR TITLE
feat(no-option): display error if no options are provided

### PR DESCRIPTION
**Pull request by sarbbottam**

### CODE REVIEW

**Comment by sarbbottam** [comment]
@ta2edchimp could you take a look?

File: test/bin/find.js [hunk 1]
```diff
@@ -40,6 +40,12 @@ describe('bin', function() {
     delete require.cache[require.resolve('yargs')]
   })
 
+  it('no option', function() {
```
Comment (ta2edchimp) [inline test/bin/find.js]: Any idea why this does not get mentioned in the PR's [corresponding travis build logs](https://travis-ci.org/sarbbottam/eslint-find-rules/jobs/127115930#L280)?

File: test/bin/find.js [hunk 2]
```diff
@@ -40,6 +40,12 @@ describe('bin', function() {
...
+    console.log = sinon.spy() // eslint-disable-line no-console
```
Comment (sarbbottam) [inline test/bin/find.js]: this is why this test title did not get printed.

File: test/bin/find.js (hunk 1 above)
Comment (sarbbottam) [inline test/bin/find.js]: fixed 
- https://travis-ci.org/sarbbottam/eslint-find-rules/jobs/127286810#L261
- https://travis-ci.org/sarbbottam/eslint-find-rules/jobs/127286811#L193
but it print the `console.logs` too.

Comment (sarbbottam) [inline test/bin/find.js]: fixed, please see 
https://travis-ci.org/sarbbottam/eslint-find-rules/jobs/127289385#L258
https://travis-ci.org/sarbbottam/eslint-find-rules/jobs/127289386#L190

Comment (ta2edchimp) [inline test/bin/find.js]: Nice solution!

**Comment by sarbbottam** [comment]
Is it good to merge?

**Comment by ta2edchimp** [comment]
Ha, yeah, I forgot to add: "LGTM"! 😜 

Pull the trigger!

**Pull request merged**